securityevent.schema.json

```diff
@@ -97,5 +102,5 @@
             "required": ["score", "severity", "vectorString"]
         }
     },
-    "required": ["eventFormatVersion", "eventId", "createdAt", "monitoringService", "severity", "confidence", "environment", "affectedResource"]
+    "required": ["eventFormatVersion", "eventId", "createdAt", "monitoringService", "severity", "confidence", "environment", "affectedResource", "actionLink"]
```

Comment:
What happen if we miss such an actionLink during the implementation or we could not provide such a link for whatever reason? Should be we really start with making this required?